### PR TITLE
chore(lanelet2_extension): change log level from error to warn for visualization

### PR DESCRIPTION
**Pull request by TakaHoribe**
## Description

change log level from error to warn for visualization

## Tests performed

run psim

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: tmp/lanelet2_extension/lib/utilities.cpp [hunk 1]
```diff
@@ -536,19 +536,18 @@ bool lineStringToPolygon(
   const lanelet::ConstLineString3d & linestring, lanelet::ConstPolygon3d * polygon)
 {
   if (polygon == nullptr) {
-    RCLCPP_ERROR_STREAM(
```
Comment (mitsudome-r) [inline tmp/lanelet2_extension/lib/utilities.cpp:539]: I think we can keep this as an ERROR since this never happens unless there is a bug in the code.

**Review by mitsudome-r — COMMENTED** [review]
Thanks for the PR. I think it's good idea to change the severity level for visualization since it shouldn't have any critical affect to the vehicle behavior.

File: tmp/lanelet2_extension/lib/utilities.cpp [hunk 2]
```diff
@@ -536,19 +536,18 @@ bool lineStringToPolygon(
...
+    RCLCPP_WARN_STREAM(
```
Comment (TakaHoribe) [inline tmp/lanelet2_extension/lib/utilities.cpp:539]:
```suggestion
    RCLCPP_ERROR_STREAM(
```

File: tmp/lanelet2_extension/lib/utilities.cpp (hunk 1 above)
Comment (TakaHoribe) [reply tmp/lanelet2_extension/lib/utilities.cpp]: Got it and done in https://github.com/autowarefoundation/autoware_common/pull/181/commits/b00adb9f01039ea803733712ccd8cd8624b92b33

**Review by mitsudome-r — APPROVED** [review]

**Pull request merged**